### PR TITLE
Update grahamcampbell guzzle factory

### PR DESCRIPTION
**Pull request by nstapelbroek**
Hey again, :wave: 

I wanted to use this package in a local project but the composer installation is blocked due to version constraints of in GrahamCampbell/Guzzle-Factory. This PR should fix that. 

### What has been done:
 - bumped graham-campbell/guzzle-factory to v3, since the API changes are minimal and did not affect the usage of the bundle I think we good :+1: 

### Notes:
- Stacked upon #143  because I needed the changes in .styleci

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^7.0",
-        "graham-campbell/guzzle-factory": "^2.1",
+        "graham-campbell/guzzle-factory": "^3.0",
```
Comment (nstapelbroek) [inline composer.json]: Come to think of it: would it be preferred to change this to: `^2.1|^3.0`?

File: tests/factories/MonitorFactory.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 $factory->define(Monitor::class, function (Faker\Generator $faker) {
     return [
-        'url' => 'http://localhost:9000',
+        'url' => 'http://localhost:'.getenv('TEST_SERVER_PORT'),
```
Comment (freekmurze) [inline tests/factories/MonitorFactory.php]: Please revert this change

**Comment by nstapelbroek** [comment]
Thank you for your review!

Should be good now :tada:

**Comment by freekmurze** [comment]
Thanks!
